peterjohnson/PETER_JOHNSON.md

```diff
@@ -0,0 +1,77 @@
+# How can I become a SDET?
+
+A better question is how can I be successful in my career? 
+What I’m going to describe here is really applicable to any career in tech!
+
+## Find your passion, so you can be dedicated
+
+I started my career in 2011 as a test technician performing manual system testing on telecommunication equipment, 
+today I work as a senior SDET, developing test automation frameworks and lead quality strategies. 
+Back in 2011, I knew very little about testing and software development and I wasn't even aware of testing as a career 
+option. However, my first job ignited a passion for learning and curiosity in computer science and testing where every 
+day was a school day, and I could learn something new. Passion is not something that just happens overnight, 
+it’s a process that takes time and sometimes obstacles get in the way, or you can get disheartened by not understanding 
+a concept at first; but the key is to persist with something a little longer even when the novelty has worn off, and 
+the challenges start. Find your passion as it will keep you dedicated, my advice would be to try many things and stick 
+with something that you are curious about and find engaging.
+
+## Develop skills that can help you solve problems
+
+Simply put, you need to develop some skills if you want to work for someone, whether that be for your employer or for 
+one of your customers. Find out what skills you bring to the table, start developing skills as soon as you can, don’t 
+fear perfection and just start and take small steps each day or week. One of the first skills I wanted to learn in 2011 
+was the ability to code in Java, and I started by finding opportunities to apply what I was learning at work and set 
+myself goals for studying so much each week. Talk to you manager, a colleague or a friend and see how people can support 
+you, ideally you want to find someone who can hold you accountable. You need to be vocal and direct about what you want 
+to learn, don’t feel it’s a weakness to discuss what you are learning with others. I’ve learned a lot from having 
+discussions with colleagues and peers by testing my ideas and understanding. Pick one skill, focus and practice it every 
+day or week. 
+Malcolm Gladwell states in his Outliers book it takes 10,000 hours of practice to master a skill, so start now!
+
+## Set goals and work towards them every week
+
+Develop a growth mindset, this is very important. Working in technology requires continuous learning, it’s one of the 
+reasons I like working in the industry as I get the opportunity to learn new technologies, approaches or methodologies 
+as part of my daily work. You must set yourself some goals and be accountable to working towards them, I write my goals 
```

Comment:
Just a suggestion, you could add a semicolon here instead of a comma.

Current: `You must set yourself some goals and be accountable to working towards them, I write my goals`

Recommended: `You must set yourself some goals and be accountable to working towards them; I write my goals`